### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by jworcestBAH**
Updates to email templates and removal of sudo check.

### CODE REVIEW

File: dataactbroker/scripts/setupEmails.py [hunk 1]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
...
+    userDb.loadEmailTemplate("DATA Act Broker - Access Approved",template,"account_approved")
 
     #Reject
-    template = "Thank you for requesting log-in credentials for the USA Spending Data Broker. Your attempt to register has been denied. If you believe this determination was made in error, please contact the USA Spending Project Management Office at DATAPMO@fiscal.treasury.gov.<br /><br />The DATA Act Implementation Team<br />DATAPMO@fiscal.treasury.gov"
-    userDb.loadEmailTemplate("USA Spending Data Broker - Access Denied",template,"account_rejected")
+    template = "Thank you for requesting log-in credentials for the DATA Act Broker. Your attempt to register has been denied. If you believe this determination was made in error, please contact the USA Spending Project Management Office at DATABroker@fiscal.treasury.gov.<br /><br />The DATA Act Implementation Team<br />DATABroker@fiscal.treasury.gov"
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Per Kristina's email, we should change all these references from the DATA Act PMO to "DATA Act Broker Helpdesk". This line in particular references the "USA Spending program management team", which is unrelated.

File: dataactbroker/scripts/setupEmails.py [hunk 2]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
...
     userDb.loadEmailTemplate("New Data Broker registration - Action Required",template,"account_creation")
 
     #User Email When finished submitting
-    template = "Thank you for registering for a user account with the USA Spending Data Broker. Your information has been sent to your agency's administrator at [EMAIL]. You will receive a response within 24 hours. <br /><br />The DATA Act Implementation Team <br />DATAPMO@fiscal.treasury.gov "
-    userDb.loadEmailTemplate("USA Spending Data Broker - Registration",template,"account_creation_user")
+    template = "Thank you for registering for a user account with the DATA Act Broker. Your information has been sent to your DATA Act Broker Help Desk at [EMAIL]. You will receive a response within 24 hours. <br /><br />The DATA Act Implementation Team <br />DATABroker@fiscal.treasury.gov "
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Change 'your DATA Act Broker Help Desk' to 'the DATA Act Broker Help Desk' so it makes grammatical sense

File: dataactbroker/scripts/setupEmails.py [hunk 3]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
     # insert email templates
 
     #Confirm
-    template = "This email address was just used to create a user account with the USA Spending Data Broker.  To continue the registration process, please click <a href='[URL]'>here</a>. The link will expire in 24 hours. <br />  <br />  If you did not initiate this process, you may disregard this email.<br /><br />The DATA Act Implementation Team<br />DATAPMO@fiscal.treasury.gov "
-    userDb.loadEmailTemplate("USA Spending Data Broker - Registration",template,"validate_email")
+    template = "This email address was just used to create a user account with the DATA Act Broker.  To continue the registration process, please click <a href='[URL]'>here</a>. The link will expire in 24 hours. <br />  <br />  If you did not initiate this process, you may disregard this email.<br /><br />The DATA Act Implementation Team<br />DATABroker@fiscal.treasury.gov "
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Change 'The DATA Act Implementation Team' to 'The DATA Act Broker Helpdesk', per Kristina's email

File: dataactbroker/scripts/setupEmails.py [hunk 4]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
...
+    userDb.loadEmailTemplate("DATA Act Broker - Registration",template,"validate_email")
 
     #Approve
-    template = "Thank you for registering for a user account with the USA Spending Data Broker. Your request has been approved by your agency's administrator. You may now log into the Data Broker portal, using the password you created at registration, by clicking <a href='[URL]'>here</a>.<br /><br /> If you have any questions, please contact your agency administrator at [EMAIL].<br /><br />The DATA Act Implementation Team<br />DATAPMO@fiscal.treasury.gov"
-    userDb.loadEmailTemplate("USA Spending Data Broker - Access Approved",template,"account_approved")
+    template = "Thank you for registering for a user account with the DATA Act Broker. Your request has been approved by your DATA Act Broker Help Desk. You may now log into the Data Broker portal, using the password you created at registration, by clicking <a href='[URL]'>here</a>.<br /><br /> If you have any questions, please contact your DATA Act Broker Help Desk at [EMAIL].<br /><br />The DATA Act Implementation Team<br />DATABroker@fiscal.treasury.gov"
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Change 'your DATA Act Broker Helpdesk' to 'The DATA Act Broker Helpdesk' (twice)

File: dataactbroker/scripts/setupEmails.py [hunk 5]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
...
+    userDb.loadEmailTemplate("DATA Act Broker - Access Denied",template,"account_rejected")
 
     #Password Reset
-    template = "You have requested your password to be reset for your account. Please click the following link <a href='[URL]'>here</a> to start the processs. The link will expire in 24 hours. <br/> <br/> If you did not request this password reset, please notify the DATA Act PMO (DATAPMO@fiscal.treasury.gov) <br /><br />The DATA Act Implementation Team <br /><br />DATAPMO@fiscal.treasury.gov"
-    userDb.loadEmailTemplate("USA Spending Data Broker - Password Reset",template,"reset_password")
+    template = "You have requested your password to be reset for your account. Please click the following link <a href='[URL]'>here</a> to start the processs. The link will expire in 24 hours. <br/> <br/> If you did not request this password reset, please notify the DATA Act PMO (DATABroker@fiscal.treasury.gov) <br /><br />The DATA Act Implementation Team <br /><br />DATABroker@fiscal.treasury.gov"
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Change "the DATA Act PMO"  to "the DATA Act Broker Helpdesk".

File: dataactbroker/scripts/setupEmails.py [hunk 6]
```diff
@@ -23,28 +23,28 @@ def setupEmails():
...
 
     #Admin Email
-    template = "This email is to notify you that the following person has requested an account for the USA Spending Data Broker:<br /><br />Name: [REG_NAME]<br /><br />Title:  [REG_TITLE]<br /><br />Agency:  [REG_AGENCY]<br /><br />Email: [REG_EMAIL]<br /><br /><br /><br />To approve or deny this user for access to the Data Broker, please click <a href='[URL]'>here</a>.<br /><br />This action must be taken within 24 hours. <br /><br />Thank you for your prompt attention.<br /><br />The DATA Act Implementation Team<br />DATAPMO@fiscal.treasury.gov"
+    template = "This email is to notify you that the following person has requested an account for the DATA Act Broker:<br /><br />Name: [REG_NAME]<br /><br />Title:  [REG_TITLE]<br /><br />Agency:  [REG_AGENCY]<br /><br />Email: [REG_EMAIL]<br /><br /><br /><br />To approve or deny this user for access to the Data Broker, please click <a href='[URL]'>here</a>.<br /><br />This action must be taken within 24 hours. <br /><br />Thank you for your prompt attention.<br /><br />The DATA Act Implementation Team<br />DATABroker@fiscal.treasury.gov"
```
Comment (kaitlin) [inline dataactbroker/scripts/setupEmails.py]: Change "The DATA Act Implementation Team" to "the DATA Act Broker Helpdesk".

File: dataactbroker/scripts/setupEmails.py (hunk 3 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

File: dataactbroker/scripts/setupEmails.py (hunk 4 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

File: dataactbroker/scripts/setupEmails.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

File: dataactbroker/scripts/setupEmails.py (hunk 5 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

File: dataactbroker/scripts/setupEmails.py (hunk 6 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

File: dataactbroker/scripts/setupEmails.py (hunk 2 above)
Comment (jworcestBAH) [inline dataactbroker/scripts/setupEmails.py]: Fixed

**Comment by jworcestBAH** [comment]
All changes made, but not pushing yet due to the demo freeze.  That said, these are DB changes so I could make these changes without committing the changed code yet, should I apply the changes to the DB before the demo?

**Comment by kaitlin** [comment]
Just wait until post demo for now. I probably won't demo the emails in the
1pm demo.

On Mon, Apr 18, 2016 at 9:46 AM, jworcestBAH <notifications@github.com>
wrote:

> All changes made, but not pushing yet due to the demo freeze. That said,
> these are DB changes so I could make these changes without committing the
> changed code yet, should I apply the changes to the DB before the demo?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/fedspendingtransparency/data-act-broker/pull/45#issuecomment-211386773>
>



-- 
Kaitlin Devine
Treasury/18F

Need to schedule a meeting? Check out my Calendly page
<https://calendly.com/kaitlin-devine>

**Comment by jworcestBAH** [comment]
Have now pushed these template changes

**Pull request merged**